CMakeLists.txt

```diff
@@ -1749,6 +1742,19 @@ elseif (APPLE)
     )
 endif()
 
+set(libvpx_export)
+if (NOT TG_OWT_VPX_PACKAGED_BUILD)
```

Comment:
```suggestion
if (NOT VPX_FOUND)
```

Reply:
@ilya-fedin, thanks for the suggestions, I just pushed improved code.